OpenRA.Mods.D2k/Traits/SpiceBloom.cs

```diff
@@ -41,12 +42,15 @@ public class SpiceBloomInfo : TraitInfo, IRenderActorPreviewSpritesInfo, Require
 		[WeaponReference]
 		public readonly string Weapon = null;
 
-		[Desc("The amount of spice to expel.")]
-		public readonly int[] Pieces = { 2, 12 };
+		[Desc("The amount of spice to expel. First number represent minimum amount. Second Maximum expel when actor is fully grow up")]
+		public readonly int[] Pieces = { 7, 20 };
 
 		[Desc("The maximum distance in cells that spice may be expelled.")]
 		public readonly int Range = 5;
 
+		[Desc("Delay between each piece spurt into air (in Ticks)")]
+		public readonly int DelayBetwenPieces = 1;
```

Comment:
Typo in Between

Reply:
OpenRA naming conventions would have this called `PieceInterval`, but IMO we should also be renaming `Pieces` to something like `Burst` (`Bursts`, `BurstInterval`)